R/stat-smooth.R

```diff
@@ -23,6 +23,9 @@
 #' @param fullrange If `TRUE`, the smoothing line gets expanded to the range of the plot,
 #'   potentially beyond the data. This does not extend the line into any additional padding
 #'   created by `expansion`.
+#' @param xseq A numeric vector of values at which the smoother is evaluated.
+#'   Internally, `NULL` defaults to a sequence of `n` equally spaced points for
```

Comment:
'When `NULL` (default), xseq is internally evaluated as a sequence of `n` equally

Reply:
spaced points for continuous data.